task_12/index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <title>Title</title>
+    <link href="https://fonts.googleapis.com/css?family=Ubuntu:300,300i,400,400i,500,500i,700,700i" rel="stylesheet">
+    <link rel="stylesheet" href="dist/css/style.css">
+    <script src="https://use.fontawesome.com/4a582f4e95.js"></script>
+    <script src="src/js/time.js"></script>
+</head>
+<body>
+<div class="wrapper">
+    <div class="content">
+        <header>
+            <div class="timeBlock">
+                <i class="fa fa-clock-o" aria-hidden="true"></i>
+                <span class="time">12 : 55 : 34</span>
+            </div>
+            <div class="logo">
+                <a class="logoLink" href="#">
+                    <img class="logoImage" src="dist/img/pizza_logo.png"
+                         alt="pizza logo">
+                </a>
+            </div>
+            <div class="logInBlock">
+                <i class="fa fa-sign-in" aria-hidden="true"></i>
+                <span class="logInText">Login / Signup</span>
+            </div>
+        </header>
+
+        <main>
+            <button class="button button__add">
+                <i class="fa fa-plus" aria-hidden="true"></i>
+                <span class="button_text">Add new pizza</span>
+            </button>
+            <section class="pizzaSection">
+                <div class="pizzaItem">
+                    <img src="dist/img/pizza_ph.jpg" alt="pizza img" class="pizzaItemImg">
+                    <span class="pizzaItemTimer">12:43:32</span>
+                    <span class="pizzaItemQueue">
+                        <span class="pizzaItemQueue__hash">#</span>
+                        <span class="pizzaItemNumberQueue">1</span>
+                    </span>
+                    <span class="pizzaItemCookedTime">
+                        <span class="pizzaItemCookedTime__text">ETA:</span>
+                        <span class="pizzaItemCookedTime__time">3</span>
+                        <span class="pizzaItemCookedTime__unit">min</span>
```

Comment:
Looks pretty strange. _3 min_ is just a text that identifies time. It may be look like following:
```html
<p class="pizzaItem__cookedTime">ETA: <time datetime="2018-02-05 18:30:45">3&nbsp;min</time></p>
```

Reply:
Tried to make the code cleaner and more understandable
Checked through the validator
Could you, please, check it again